src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts

```diff
@@ -47,10 +47,10 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
   HealthSub: Subscription;
   BucketSub: Subscription;
   queriesResults: any = {
```

Comment:
```suggestion
  queriesResults: {[key: string]: []} = {
```

Reply:
I personally prefer to go with, `Record<string, []>`
I think the Record type it is way more readable and allows for some more flexibility on the defined types